AmplifyPlugins/DataStore/AWSDataStoreCategoryPluginTests/Storage/StorageEngineTestsBase.swift

```diff
@@ -95,7 +95,7 @@ class StorageEngineTestsBase: XCTestCase {
     func deleteModelSynchronous<M: Model>(modelType: M.Type,
                                           withId id: String,
                                           where predicate: QueryPredicate? = nil,
-                                          timeout: TimeInterval = 1) -> DataStoreResult<M?> {
+                                          timeout: TimeInterval = 10) -> DataStoreResult<M?> {
```

Comment:
this change isn't needed and was because i was stepping through tests that increasing it by a factor helped debug, since i would continue on breakpoints within 10 seconds. I opted to keep this here so that next time we have to debug, we don't have to change it again, and revert it back, unless there's a good reason to keep it 1 second. Keeping it 1s is probably better representation of the time it takes to perform this method though. let me know if u prefer reverting this change